main.py

```diff
@@ -0,0 +1,88 @@
+"""Only user inputs with predefined list of choices."""
+
+from event import *
+from manager import *
+from datetime import date
+
+
+class Main(object):
+
+    manager_instance = Manager()
+
+    while (1):
+        print "1: add_event  2: read_event  3: update_event  4: delete_event  5: upcoming_and_completed events  " \
+              "6: list_event_by_date_or_by_city  7: list_events_in_date_range  8: Exit"
+
+        choice = int(raw_input("choice:"))
+
+        if choice == 1:
+            event_instance = Event()
+            event_instance.set_name(raw_input("enter event name :"))
+            event_instance.set_city(raw_input("enter city :"))
+            event_instance.set_info(raw_input("enter info :"))
+            event_instance.set_date(str(date(int(raw_input("enter year(yyyy) :")),
+                                             int(raw_input("enter month(mm) :")),
+                                             int(raw_input("enter day(dd) :")))))
+            manager_instance.add_event(event_instance)
+
+        elif choice == 2:
+            manager_instance.read_event_by_id(raw_input("enter the event id :"))
```

Comment:
manager should not print the records. It should return the records. If you print them in the manager class, later if we ask you implement the same as a Web app, you wont be able to reuse the existing code base.